drivers/spi/spi-intel.c

```diff
@@ -1113,12 +856,19 @@ static int intel_spi_init(struct intel_spi *ispi)
 		return -EINVAL;
 	}
 
+	if ((ispi->swseq_erase || !erase_64k) && !ispi->swseq_enabled)
```

Comment:
Also my general comment, are we sure what ispi->swseq_reg means? are we sure that when that is true hwseq is fully available?